x-pack/plugins/security_solution/public/overview/components/detection_response/cases_by_status/cases_by_status.test.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { BarChartComponentProps } from '../../../../common/components/charts/barchart';
+import { useQueryToggle } from '../../../../common/containers/query_toggle';
+import { TestProviders } from '../../../../common/mock';
+import { CasesByStatus } from './cases_by_status';
+jest.mock('../../../../common/components/link_to');
+jest.mock('../../../../common/containers/query_toggle');
+jest.mock('./use_cases_by_status', () => ({
+  useCasesByStatus: jest.fn().mockReturnValue({
+    closed: 1,
+    inProgress: 2,
+    isLoading: false,
+    open: 3,
+    totalCounts: 6,
+    updatedAt: new Date('2022-04-08T12:00:00.000Z').valueOf(),
+  }),
+}));
+jest.mock('../../../../common/lib/kibana', () => {
+  const actual = jest.requireActual('../../../../common/lib/kibana');
+  return {
+    ...actual,
+    useNavigation: jest.fn().mockReturnValue({
+      getAppUrl: jest.fn(),
+      navigateTo: jest.fn(),
+    }),
+  };
+});
+jest.mock('../../../../common/components/charts/barchart', () => ({
+  BarChart: jest.fn((props: BarChartComponentProps) => (
+    <div data-test-subj="barChart" {...props} />
+  )),
+}));
+
+describe('CasesByStatus', () => {
+  const mockSetToggle = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    (useQueryToggle as jest.Mock).mockReturnValue({
+      toggleStatus: true,
+      setToggleStatus: mockSetToggle,
+    });
```

Comment:
NIT: same, this mocks could be defined outside the test.
if you don't want to mess with the default mock at line 82, you can use `mockReturnValueOnce` so it will only change the return value the one call.